### ISSUE TITLE
Brittle approach to playing collision sound

### ISSUE THREAD

**Issue by pixelzoom** [phetsims/my-solar-system#291]
While investigating #290, I discovered this in SolarSystemCommonScreenView:

```ts
    model.bodies.forEach( body => {
      body.collidedEmitter.addListener( () => {
        this.bodySoundManager.playBodyRemovedSound( 3 ); // Plays the collision sound instead of body index
      } );
    } );
```

Magic number `3` makes this brittle -- why 3? 

A better approach would be to add `bodySoundManager.playBodyCollidedSound()`.

**Comment by pixelzoom** [comment]
This code in BodyManager.ts answers my question about "why 3".   It also confirms that this is a brittle approach.  If more bodies were ever added, the above code would break.

```ts
const removalSounds = [
  Bodies_Collide_Absorb_2_to_1_mp3,
  Bodies_Collide_Absorb_3_to_2_mp3,
  Bodies_Collide_Absorb_4_to_3_mp3,
  Collision_Sound_mp3
];
```

**Comment by AgustinVallejo** [comment]
Will fix right away

**Comment by pixelzoom** [comment]
I pushed a fix in the above commit. Compared to previous dev versions, and it seems to do the job.  @AgustinVallejo please review, close if OK.

**Comment by AgustinVallejo** [comment]
Seems about right! Closing

**Comment by pixelzoom** [comment]
Hang on... I have a couple of additional improvements coming.

**Comment by pixelzoom** [comment]
In https://github.com/phetsims/solar-system-common/commit/3a2456b82efab6676c5efe11d3354244156386af, I renamed the following fields in BodySoundManager, for consistency with their associated method names:
* bodyNumberSoundClips => bodyAddedSoundClips
* removalSoundClips => bodyRemovedSoundClips
* collidedSoundClip => bodyCollidedSoundClip

In the other 2 commits, I changed the `bodyNumber` parameter to `playBodyAddedSound` and `playBodyRemovedSound`. This is not in fact a bodyNumber, because bodieds are numbered starting at 1.  It's actually the number of bodies in `activeBodies.`.

I'll go ahead and re-close this, but @AgustinVallejo feel free to take a look.
